dist/well-known.js

```diff
@@ -115,6 +116,54 @@ return Promise.all([
       return data;
     });
   }),
+  // FedCM
+  parseResponse('/.well-known/web-identity', r => {
+    return r.json().then(data => {
+        let result = {
+            provider_urls: Array.isArray(data.provider_urls) && data.provider_urls.length > 0 ? data.provider_urls : [],
+            accounts_endpoint: data.accounts_endpoint || null,
+            login_url: data.login_url || null,
+            valid: Array.isArray(data.provider_urls) && data.provider_urls.length > 0
+        };
+        return result;
+    }).catch(error => {
+        return {
+            error: `Failed to parse JSON: ${error.message}`,
+            valid: false
+        };
+    });
+  }),
+  // Passkey
+  parseResponse('/.well-known/passkey-endpoints', r => {
+    return r.json().then(data => {
+        let result = {
+            enroll: data.enroll || null,
+            manage: data.manage || null,
+            valid: typeof data.enroll === 'string' && typeof data.manage === 'string'
```

Comment:
Indeed seems like `valid` duplicates the other two.
```suggestion
            enroll: data.enroll || null,
            manage: data.manage || null
```